### PR TITLE
GLTFLoader: Ensure InterleavedBuffer instances are shared.

### PR DESCRIPTION
**Pull request by donmccurdy**
/cc @takahirox

### CODE REVIEW

**Comment by takahirox** [comment]
Looks ok to me.

File: examples/js/loaders/GLTFLoader.js [hunk 1]
```diff
@@ -1536,11 +1536,20 @@ THREE.GLTFLoader = ( function () {
 			// The buffer is not interleaved if the stride is the item size in bytes.
 			if ( byteStride && byteStride !== itemBytes ) {
 
-				// Use the full buffer if it's interleaved.
-				array = new TypedArray( bufferView );
+				var ibCacheKey = 'InterleavedBuffer:' + accessorDef.bufferView + ':' + accessorDef.componentType;
+				var ib = parser.cache.get( ibCacheKey );
+
+				if ( !ib ) {
```
Comment (takahirox) [inline examples/js/loaders/GLTFLoader.js]: You need a space after `ib` for the code style, very minor stuff tho.

Comment (donmccurdy) [reply examples/js/loaders/GLTFLoader.js]: I'm not sure what you mean — like `! ib` instead of `!ib`?

Comment (WestLangley) [reply examples/js/loaders/GLTFLoader.js]:
```javascript
if ( ! ib ) {
```

Comment (donmccurdy) [reply examples/js/loaders/GLTFLoader.js]: Thanks, done.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**